target_hdfs/__init__.py

```diff
@@ -197,7 +197,7 @@ def consumer(receiver):
                 write_file_for_current_stream = True
```

Comment:
```suggestion
                write_file_for_current_stream = `bool(current_stream_name)`
```
This will explain a bit more about what's happening.
It's weird to have `write_file_for_current_stream = True` but not write do HDFS.